motion_computation/src/motion_computation_worker.cpp

```diff
@@ -133,7 +133,7 @@ cav_msgs::ExternalObjectList MotionComputationWorker::predictionLogic(cav_msgs::
    
   }//end for-loop
 
-  return list;
+  obj_pub_(list);
```

Comment:
If you are going to publish from here why do we need the indirection of the motionPredictionCallback function.